### ISSUE TITLE
Error: Invalid package.

### ISSUE THREAD

**Issue by tobeycodes** [alwaysblank/blade-generate#1]
`wp package install benspants/blade-cache-management`

**Comment by alwaysblank** [comment]
There was a spelling error in the README. Please try: 

`wp package install alwaysblank/blade-cache-management`

**Comment by jrstaatsiii** [comment]
I'm also experiencing this issue, with the corrected command.

**Comment by tobeycodes** [comment]
https://github.com/wp-cli/package-index

**Comment by alwaysblank** [comment]
Hi folks, sorry for the delay.

I've submitted this to the wp-cli package index as per the @schrapel link, but unfortunately it looks like submissions are on indefinite hold: https://github.com/wp-cli/package-index/pull/90#issuecomment-264880112

For now, the best approach is probably to just install it directly from github:
`wp package install git@github.com:alwaysblank/blade-generate.git`

I've updated the readme to reflect this approach, and tested it locally.

I'm closing this issue as the above suggestion (and change in the readme) should resolve it, but if you run into another other issues, let me know.
